crates/bevy_ecs/src/system/function_system.rs

```diff
@@ -20,7 +20,19 @@ use super::{In, IntoSystem, ReadOnlySystem, SystemBuilder};
 #[derive(Clone)]
 pub struct SystemMeta {
     pub(crate) name: Cow<'static, str>,
+    /// The set of component accesses for this system. This is used to determine
+    /// - soundness issues (e.g. multiple [`SystemParams`](`SystemParam`) mutably accessing the same component)
```

Comment:
```suggestion
    /// - soundness issues (e.g. multiple [`SystemParam`]s mutably accessing the same component)
```

I'd go for this but it doesn't matter